includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -41,7 +41,19 @@ public function __construct() {
 		$this->total_label     = ! empty( $this->stripe_settings['statement_descriptor'] ) ? WC_Stripe_Helper::clean_statement_descriptor( $this->stripe_settings['statement_descriptor'] ) : '';
 
 		$this->total_label = str_replace( "'", '', $this->total_label ) . apply_filters( 'wc_stripe_payment_request_total_label_suffix', ' (via WooCommerce)' );
+	}
 
+	/**
+	 * Returns the suffix set for the express payment method titles.
+	 *
+	 * @return mixed
```

Comment:
Shouldn't this method always return a string?

```suggestion
	 * @return string
```